python/src/scipp/runtime_config.py

```diff
@@ -77,11 +77,11 @@
     },
     # The colors for each dataset member used in table and show functions
     "colors": {
-        "coord": "#dde9af",
+        "coords": "#dde9af",
         "data": "#ffe680",
         "labels": "#afdde9",
-        "attr": "#ff8080",
-        "mask": "#cccccc",
+        "attrs": "#ff8080",
+        "masks": "#d2b4de",
```

Comment:
Why the change to the inconsistent plural, and change of the color for masks?

Reply:
oh, the masks color i changed because it looked very similar to the headers in the table where it says `Values` and `Variances`, but it's not that important.

Changing to plural for masks and attrs makes sense to me, because it's like labels, there can be more than one, and the attributes of the `Dataset` are `.attrs` and `.masks`.
It was the same reason for the coords: the attribute is `.coords`, although one could argue that you will only have one coordinate in a table. However, you can have more than one in a `show` output.

Finally, it makes it possible to write very succint and generic code, however, it is now more inflexible since if you change this back, I can no longer use the `getattr`s [here](https://github.com/scipp/scipp/blob/f08345964be6266d2816a65c20d9f0b0f9aa058a/python/src/scipp/table.py#L217)

Reply:
> Changing to plural for masks and attrs makes sense to me, because it's like labels, there can be more than one, and the attributes of the Dataset are .attrs and .masks.
It was the same reason for the coords: the attribute is .coords, although one could argue that you will only have one coordinate in a table. However, you can have more than one in a show output.

The attributes are all plural, because it returns the dict of coords, labels, masks, or attributes. When talking about an individual entry (see `setCoord`, `setLabels`, `setMask`, `setData`), only "labels" is plural, because for all the others the word collectively refers to an array (coord is an array of values, a mask is an array of boolean values, data is an array), whereas "the label is an array" is weird, so this one has the plural "s". If anything, I would consider removing the `s` from labels, everywhere.

Reply:
you could also just think of it as "this is the color for masks", instead of "this is the color for a mask" and it would still make sense.
I understand the point about `setMask` but then `setData` is also inconsistent, being a plural word.
Anyway, these colors are only used on the python side, so I thought it made sense because a python user will never see the C++ side of things.

Reply:
I'm ok with that, but could you change the masks back to grey? I feel that is more intuitive, e.g., visually similar to grey-out GUI elements or rows in a table.